src/material-experimental/mdc-chips/chips.scss

```diff
@@ -16,6 +16,9 @@
     // were disabled via the `NoopAnimationsModule`, the element won't have a transition and
     // `transitionend` won't fire. We work around the issue by assigning a very short transition.
     transition-duration: 1ms;
+
+    // Disables the chip enter animation.
+    animation: none;
```

Comment:
Are we sure that mdc-chips doesn't use the animation events similar to how it uses te transition events? If not this should be ok, otherwise we may have to go with "almost disabled" like we did above

Reply:
From what I could tell it wasn't using the enter animation for anything.